cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -764,10 +764,13 @@ endif()
 #---Check for fitsio-------------------------------------------------------------------
 if(fitsio OR builtin_cfitsio)
   if(builtin_cfitsio)
-    set(cfitsio_version 3.280)
+    set(cfitsio_version 3.450)
     string(REPLACE "." "" cfitsio_version_no_dots ${cfitsio_version})
     message(STATUS "Downloading and building CFITSIO version ${cfitsio_version}")
+    # cfitsio 3.45 depends on curl libraries 
+    find_package(CURL QUIET)
```

Comment:
What happens if CURL is not found?

Reply:
I guess the Windows build already shows it:
```
14:55:47 CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
14:55:47 Please set them or make sure they are set and tested correctly in the CMake files:
14:55:47 CURL_LIBRARY
14:55:47     linked by target "FITSIO" in directory C:/build/workspace/root-pullrequests-build/root/graf2d/fitsio
14:55:47 
14:55:47 -- Configuring incomplete, errors occurred!
```

Reply:
Ok, lets require it strictly, since builtin CFITSIO will not compile ( I see inside tar.gz, curl is strictly required in its CMakelists.txt)

Reply:
I talked with Bertrand, and curl should not be required strictly as well how it is done for Windows (e.g windows doesn't have curl installed).
We just need to know in case if curl was enabled for CFITSIO for Linux builds (basically it is always like this, since curl is usually installed on Linux systems) and add CURL libraries in a list CFITSIO libraries.